### ISSUE TITLE
Rebuilding sdk

### ISSUE THREAD

**Issue by Nicolo-sudo** [BelledonneCommunications/linphone-android#821]
Hi there, is possible to enable H263 codec by rebuilding the linphone-sdk?
If I build the sdk by passing the option "-DENABLEH263=ON" then I have to change something in code or is enough?
How can I import the new sdk to an Android Studio Project?

Thanks in advance!

**Comment by Viish** [comment]
There is no -DENABLE_H263 but instead it is -DENABLE_FFMPEG as it is ffmpeg that brings the encoder/decoder for H263.
Once the SDK is built with ffmpeg no you don't have to change anything in the code.
To use you own SDK build, check the README there is a section about how to do that.

**Comment by Nicolo-sudo** [comment]
Hi, I tried to enable the H263 codec by following your instructions but ended up getting the error: "No download info given for 'EP_ffmpegandroid' and its source directory"

I have cloned the linphone-sdk with all submodules, then I launched the command: 
"mkdir build && cd build && cmake -DLINPHONESDK_PLATFORM=

ANDROID -DENABLE_NON_FREE_CODECS=ON -DENABLE_FFMPEG=ON ..

Then I run:
"cmake --build ."

I have attached the logs in this post.



I read here (https://github.com/BelledonneCommunications/linphone-android/issues/457) 
[logs.zip](https://github.com/BelledonneCommunications/linphone-android/files/3823669/logs.zip)
that there is a problem with the FFMPEG so this issue has been fixed? How can I solve this problem?
